pages/open_cas_linux/guide_monitoring_json.md

```diff
@@ -0,0 +1,497 @@
+      "Page": 259,
+      "Fraction": 10000
+    },
+    "Reads from cache(s)": {
+      "Page": 0,
+      "Fraction": 0
+    },
+    "Writes from cache(s)": {
+      "Page": 259,
+      "Fraction": 10000
+    },
+    "Total from/to cache(s)": {
+      "Page": 259,
+      "Fraction": 10000
+    },
+    "Reads from exported object(s)": {
+      "Page": 259,
+      "Fraction": 10000
+    },
+    "Writes from exported object(s)": {
+      "Page": 0,
+      "Fraction": 0
+    },
+    "Total from/to exported object(s)": {
+      "Page": 259,
+      "Fraction": 10000
+    }
+  },
+  "Errors": {
+    "Core read errors": {
+      "Page": 0,
+      "Fraction": 0
+    },
+    "Core write errors": {
+      "Page": 0,
+      "Fraction": 0
+    },
+    "Core total errors": {
+      "Page": 0,
+      "Fraction": 0
+    },
+    "Cache read errors": {
+      "Page": 0,
+      "Fraction": 0
+    },
+    "Cache write errors": {
+      "Page": 0,
+      "Fraction": 0
+    },
+    "Cache total errors": {
+      "Page": 0,
+      "Fraction": 0
+    },
+    "Total errors": {
+      "Page": 0,
+      "Fraction": 0
+    }
+  }
+}
+```
+
+# List caches
+
+Checking CAS devices with casadm list caches mentioned in [**Monitoring CAS**](/guide_monitoring.html).
+
+```console
+[root@localhost json]# casadm -L
+\type    id   disk             status    write policy   device
+cache   1    /dev/nvme3n1p4   Running   wt             -
+├core   1    /dev/nvme0n1p3   Active    -              /dev/cas1-1
+└core   2    /dev/nvme0n1p4   Active    -              /dev/cas1-2
+```
+
+# Example list caches request
+
+```json
+{
+  "header": {
+      "command": "cache.get_list"
+  },
+  "params": {
+  }
+}
+```
+
+# Example list caches response
+
+**Example List caches response structure:**
+
+-   for each cache instance:
+    -   Cache details
+    -   for each core in cache:
+        -   Core details
+```json
+[
+  {
+    "Cache": {
+      "Cache id": 1,
+      "Cache device": "/dev/disk/by-id/nvme-nvme.8086-50484b53373438323030335433373541474e-494e54454c20535344504544314b3337354741-00000001-part4",
+      "Core(s) id(s)": [
+        1,
+        2
+      ],
+      "Cache details": {
+        "Attached": true,
+        "Status": "stopping",
+        "Size [cache lines]": 5183520,
+        "Inactive cores": {
+          "Occupancy": {
+            "Page": 0,
+            "Fraction": 0
+          },
+          "Clean": {
+            "Page": 0,
+            "Fraction": 0
+          },
+          "Dirty": {
+            "Page": 0,
+            "Fraction": 0
+          }
+        },
+        "Occupancy [cache lines]": 518,
+        "Dirty [cache lines]": 0,
+        "Dirty for [s]": 0,
+        "Initially dirty [cache lines]": 0,
+        "Cache mode": "wt",
+        "Pass-Trough fallback statistics": {
+          "IO errors count": 0,
+          "Status": false
+        },
+        "Cleaning policy": "alru",
+        "Promotion policy": "always",
+        "Cache line size [KiB]": 4096,
```

Comment:
4096 KiB is a lot for a cache line size ;)

Reply:
Changed for 4096 B :)